src/data/search.tsx

```diff
@@ -24,17 +26,34 @@ function indexAkn(html: Document): IndexedObject[] {
     let title = (titleSelector ? titleSelector.textContent : "") || "";
 
     searchData.push({
-      item: {
-        id: section.id,
-        title: title,
-        children: [],
-      },
+      item: '',
+      title: title,
       id: section.id,
       content: title.toLocaleLowerCase() + text.join(' ').toLocaleLowerCase(),
     });
   });
 
-  return searchData;
+  const lunrSearchData = lunr(function () {
+    // @ts-ignore
+    this.ref('id');
```

Comment:
There must be only one ref. This is the field that is used as the id of the search result.